contributor/03-contributor-ethos-article.md

```diff
@@ -21,56 +21,51 @@ knocking or ringing the door bell even if the door is open. Likewise in InnerSou
 as a visitor you won't be able (or invited) to commit directly to any
 code repository.
 
-Instead, when visiting your neighbors you're likely to at least knock on their
-door. Likewise in InnerSource after making your changes to the codebase you'll
+Instead, after making your changes to the codebase you'll
```

Comment:
I actually liked the conversational part here, it is followed throughout the rest of the articles and, if I remember correctly, referenced to later.

Reply:
Knocking is already mentioned on line 20, so it may be redundant to mention here again.

Reply:
I agree, closing this one then.